docs/troubleshoot/app-framework/zss-error-codes.md

```diff
@@ -0,0 +1,371 @@
+
+  **Reason:**
+
+  Caching Service settings are gateway host `<Gateway-host>`, port `<Gateway-port>`.
+
+  **Action:**
+
+  No action required.
+
+### ZWES0064I
+
+  Caching Service not configured
+
+  **Reason:**
+
+  Caching Service not configured.
+
+  **Action:**
+
+  No action required.
+### ZWES1100I
+
+  Product Registration is enabled.
+
+  **Reason:**
+
+  Product Registration is enabled.
+
+  **Action:**
+
+  No action required.
+
+### ZWES1101I
+
+  Product Registration is disabled.
+
+  **Reason:**
+
+  Product Registration is disabled.
+
+  **Action:**
+
+  No action required.
+
+### ZWES1102I
+
+  Product Registration successful.
+
+  **Reason:**
+
+  Product Registration successful.
+
+  **Action:**
+
+  No action required.
+
+### ZWES1600I
+
+  JWT will be configured using JWK URL '%s'
+
+  **Reason:**
+
+  JWT will be configured using JSON Web Key(JWK) at URL <url>.
+
+  **Action:**
+
+  No action required.
+
+### ZWES1601I
+
+  Server is ready to accept JWT `with`(or `without`) fallback to legacy tokens
+
+  **Reason:**
+
+  Server is ready to accept JWT `with` or `without` fallback to legacy tokens.
+
+  **Action:**
+
+  No action required.
+
+## ZSS error messages
+
+
+### ZWES1006E
+
+  Error while parsing plugin definition file '%s': '%s'.
+
+  **Reason:**
+
+  An error occurred while parsing `<plugin-definition-file>`: `<error-details>`.
+  
+
+  **Action:**
+
+  Check `<error-details>` and fix the error by editing `<plugin-definition-file>`.
+
+### ZWES1034E
+
+  Server startup problem: Address '%s' not valid.
+
+  **Reason:**
+
+  IP address is not valid.
+
+  **Action:**
+
+  Use valid IP address, e.g. `0.0.0.0`.
+
+### ZWES1036E
+
+  Server startup problem: Ret='%d', res='0x%x'
+
+  **Reason:**
+
+  Server has failed to start.
+
+  **Action:**
+
+  If the next message is `ZWES1037E` then refer [ZWES1037E](###ZWES1037E).
+
+### ZWES1037E
+
+  This is usually because the server port '%d' is occupied. Is ZSS running twice?
+
+  **Reason:**
+
+  ZSS port number is already occupped. 
+
+  **Action:**
+
+  Check if another ZSS instance is already running, or chose another free port number.
+
+### ZWES1065E
+
+  Failed to configure https server, check agent https settings
+
+  **Reason:**
+
+  Failed to configure https server. 
+
+  **Action:**
+
+  Check agent https settings.
+
+
+## ZSS warning messages
+
+### ZWES1000W
+
+  Privileged server name not provided, falling back to default.
+
+  **Reason:**
+
+  Privileged server name not configured in `zowe.yaml`.
+
+  **Action:**
+
+  If your privileged server name is `ZWESIS_STD` then no action required. Otherwise set `components.zss.crossMemoryServerName` property in `zowe.yaml` to the correct name.
+
+### ZWES1005W
+
+  Plugin ID was not found in '%s'
+
+  **Reason:**
+
+  `pluginId` property wasn't found in `<path-to-pluginDefinition.json>` file.
+
+  **Action:**
+
+  Add `pluginId` property into `<path-to-pluginDefinition.json>` file.
```

Comment:
Or if not a developer, contact the person who made your plugin because its broken.